modules/framework/galasa-parent/dev.galasa.framework/src/test/java/dev/galasa/framework/mocks/MockRun.java

```diff
@@ -52,6 +53,26 @@ public MockRun(
         this.isRunLocal = isRunLocal;
         this.gherkinUrl = gherkinUrl;
     }
+
+    public MockRun(
+        String testBundleName, 
+        String testClassName, String testRunName , 
+        String testStream, String testStreamOBR, 
+        String testStreamRepoUrl, String requestorName, 
+        boolean isRunLocal , String gherkinUrl,
+        String group
+    ) {
+        this.testBundleName = testBundleName;
+        this.testClassName = testClassName ;
+        this.testRunName = testRunName;
+        this.testStream = testStream;
+        this.testStreamOBR = testStreamOBR;
+        this.testStreamRepoUrl = testStreamRepoUrl;
+        this.requestorName = requestorName;
+        this.isRunLocal = isRunLocal;
+        this.gherkinUrl = gherkinUrl;
+        this.group = group;
```

Comment:
If this constructor is just adding `group`, we could reduce the duplicate code by calling the other constructor and then setting group, or have the existing constructor call this one with group set to null